### PR TITLE
New package: podman-1.0.0

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/cri-o/template [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	vinstall bin/conmon 755 usr/libexec/crio
+	vinstall bin/pause 755 usr/libexec/crio
+	make install.man PREFIX="${DESTDIR}/usr"
+	make install.config PREFIX="${DESTDIR}/usr" DESTDIR="${DESTDIR}"
+	vmkdir usr/share/examples/"${pkgname}"
+	vcopy contrib/cni usr/share/examples/"${pkgname}"
+}
+
+cri-o-devel_package() {
+	short_desc+=" - development tools"
```
Comment (maxice8) [inline srcpkgs/cri-o/template]: should depend on mainpkg

Comment (CameronNemo) [reply srcpkgs/cri-o/template]: done

**Comment by CameronNemo** [comment]
at least a few rough edges remain:

* apparmor won't work without #5637
* you can't pull any containers without buildah installed (PRs coming)
* you must configure a cni bridge network with the name 'podman' (example conf and INSTALL.msg to be added)

**Comment by CameronNemo** [comment]
This is pretty much finished from my perspective. @jcgruenhage @knusbaum please test and / or review.

**Comment by CameronNemo** [comment]
@jcgruenhage no worries, better to get it right than get it fast.

**Pull request merged**